.github/workflows/dev-test.yml

```diff
@@ -24,6 +24,9 @@ jobs:
           - os: "ubuntu-latest"
             node: "12"
             ENABLE_CODE_COVERAGE: true
+        exclude:
+          - os: "macos-latest"
+            node: "13"
```

Comment:
Do you exclude it simply because macos tests take longer and are flakier or is there something else?

Reply:
Github Actions limit Maximum concurrent macOS jobs to 5, https://help.github.com/en/actions/getting-started-with-github-actions/about-github-actions#usage-limits